File: java/1480_Running_Sum_of_1d_Array.java
class Solution {
    public int[] runningSum(int[] nums) {
        List<Integer> output = new ArrayList<Integer>();
        int sum = 0;
        for (int num : nums) {
            sum = sum + num;
            output.add(sum);
        }
        return output.stream().mapToInt(i -> i).toArray();
    }
}

File: kotlin/1480_Running_Sum_of_1d_Array.kt
class Solution {
    fun runningSum(nums: IntArray): IntArray {
        var output: IntArray = IntArray()
        var sum = 0
        for (i=0;i<nums.size;i++) {
            sum = sum + num;
            output.set(i, sum)
        }
        return output;
    }
}